docs/concepts/withdrawals.md

```diff
@@ -66,8 +66,10 @@ of Capella), there's no mechanism for recycling the validator ID that has been e
 From the first Capella slot, block proposers provide up to 16 withdrawals per
 proposed block.
 
-Proposers start from validator 1, and find validators that qualify: must have an
-Ethereum address as a withdrawal address, must have an excess balance, or have exited.
+Proposers start with validator 1 and search for validators that meet any of these criteria:
+- have an Ethereum address as a withdrawal address,
+- posses an excess balance,
+- have exited.
```

Comment:
```suggestion
- have an Ethereum address as a withdrawal address,
- posses an excess balance,
- have exited.
```

Please review the list guidelines from the Microsoft Style Guide: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists. Items should start with an uppercase letter and end with the same punctuation.